docs/UserGuide.md

```diff
@@ -934,8 +942,4 @@ For example:
 
 This glossary provides definitions for the special terms used in this user guide.
```

Comment:
Are there any terms we plan on putting in the glossary?

Reply:
Not sure... Perhaps it can be removed at the end if no one adds anything?